plutus_bench/mock.py

```diff
@@ -308,6 +310,7 @@ def evaluate_tx(self, tx: Transaction) -> Dict[str, ExecutionUnits]:
         for invocation in script_invocations:
             # run opshin script if available
             if self.opshin_scripts.get(invocation.script) is not None:
+                raise NotImplementedError("This code never seems to be reached")
```

Comment:
this is correct. this feature was added to allow python debuggers to execute the relevant code when invoking the code programmatically from other python code - which is not goal of the design of plutus-bench anymore